libs/openchallenges/team/src/lib/team.component.scss

```diff
@@ -27,4 +27,10 @@
 
 .description {
   padding-bottom: 180px;
-}
\ No newline at end of file
+}
+
```

Comment:
<img width="400" alt="Screen Shot 2023-08-03 at 4 00 58 PM" src="https://github.com/Sage-Bionetworks/sage-monorepo/assets/73901500/6f55deff-ab18-4439-94f6-731753846290">

The roles are not aligned since they have different rows.

```suggestion
.card-roles {
  flex-grow: 1;
}
```
Can you make `roles` expand so that the texts of cards will aligned?

Reply:
Nice! I've implemented this.